src/constants/errorWrapperOptions.ts

```diff
@@ -0,0 +1,22 @@
+import { RouteHandlerErrorProperties } from '../types';
+
+export const DEFAULT_CUSTOM_LOG_FUNCTION = ({
+  method,
+  originalUrl,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
+  statusCode,
```

Comment:
You don't need to destructure statusCode 
```suggestion
```

Reply:
I'm making it available to the custom function, just not using it in the default.

Reply:
oh nvm I see why it can be omitted here